pkg/cmd/run/root.go

```diff
@@ -116,6 +135,10 @@ var runCmd = &cobra.Command{
 
 		time.Sleep(time.Second * time.Duration(2))
 
+		spinner.Message("Starting functions")
+
+		time.Sleep(time.Second * time.Duration(1))
```

Comment:
Are these two sleeps in a row necessary?

Reply:
Will look at doing sub commands like the suggestion above. Short answer no 👍